<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQClass</class>
 <widget class="QWidget" name="QQClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QWidget" name="login_bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>331</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>265</y>
      <width>240</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(65,185,255);
    color:white;
    border:0px;
}
QPushButton:hover
{
    background-color:rgb(85,175,245);
}

QPushButton:pressed
{
    background-color:rgb(55,175,235);
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 12pt &quot;黑体&quot;;
	background-color:transparent;
    color:white;
    border:0px;
}
QPushButton:hover
{
    background-color:rgb(245,20,0);
}

QPushButton:pressed
{
    background-color:rgb(215,45,0);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:transparent;
    color:white;
    border:0px;
}
QPushButton:hover
{
    background-color:rgba(230,230,230,150);
}

QPushButton:pressed
{
    background-color:rgb(240,240,240);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_account_bg" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QLineEditEx" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;微软雅黑&quot;;
border-color:transparent;
background-color: transparent;
border:0 px;</string>
     </property>
     <property name="placeholderText">
      <string>QQ号码/手机/邮箱</string>
     </property>
    </widget>
    <widget class="Line" name="line1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>28</y>
       <width>240</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 216, 216);</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
image: url(:/QQ/Resources/ico_pgun.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="acc_combox_btn">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>0</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:transparent;
    color:rgb(55,55,55);
    border:0px;
}</string>
     </property>
     <property name="text">
      <string>﹀</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_pwd_bg" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QLineEditEx" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>0</y>
       <width>190</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
   background-color:transparent;
   border:0 px;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/QQ/Resources/ico_lock.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="acc_combox_btn_2">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/QQ/Resources/ico_board.png);
    color:rgb(55,55,55);
    border:0px;
}
background-image: url(:/QQ/Resources/ico_board.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>28</y>
       <width>240</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 4);</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_top_bg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QLabel" name="top_flow_bg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>420</width>
       <height>121</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.180867 rgba(157, 36, 23, 255), stop:1 rgba(163, 255, 255, 249));</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/QQ/Resources/qqio.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>5</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 24pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>QQ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_set">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>30</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/QQ/Resources/ico_set.png);
	background-color:transparent;
    color:white;
    border:0px;
}
QPushButton:hover
{
    background-color:rgba(230,230,230,150);
}

QPushButton:pressed
{
    background-color:rgb(240,240,240);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="opt_bg" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="auto_login_checkBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>8</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>自动登录</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="auto_login_checkBox_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>8</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>8</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>找回密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>85</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/QQ/Resources/QQ_Penguin_tencent_128px_500133_easyicon.net.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;style&gt; a {text-decoration: none} &lt;/style&gt;&lt;a style='color: grey;' href=&quot;https://ssl.zc.qq.com/v3/index-chs.html&quot;&gt;注册账号</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;
background-color: rgb(194, 194, 194);</string>
    </property>
    <item>
     <property name="text">
      <string>907809403</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>778804656</string>
     </property>
    </item>
   </widget>
   <zorder>btn_login</zorder>
   <zorder>widget_account_bg</zorder>
   <zorder>widget_pwd_bg</zorder>
   <zorder>widget_top_bg</zorder>
   <zorder>btn_close</zorder>
   <zorder>btn_min</zorder>
   <zorder>opt_bg</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>listWidget</zorder>
  </widget>
  <widget class="QWidget" name="set_bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>154</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>不使用代理</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTTP代理</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>使用浏览器设置</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>set_bg</zorder>
  <zorder>login_bg</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLineEditEx</class>
   <extends>QLineEdit</extends>
   <header location="global">qlineeditex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QQ.qrc"/>
 </resources>
 <connections/>
</ui>
